### PR TITLE
joypad

### PR DESCRIPTION
**Pull request by AlvaroBurnett**

### CODE REVIEW

File: content/Joypad_Input.md [hunk 1]
```diff
@@ -31,17 +31,17 @@ to the SNES. See SGB description for details.
 ### INT 60 - Joypad Interrupt
 
 The Joypad interrupt is requested when any of the above Input lines changes
```
Comment (ISSOtm) [inline content/Joypad_Input.md:33]: "Input lines" should be changed to "bits 0-3" to better hint that bit 4 and/or 5 need to select them.

File: content/Joypad_Input.md [hunk 2]
```diff
@@ -31,17 +31,17 @@ to the SNES. See SGB description for details.
...
-from High to Low. This happens when a key is
-pressed (provided that the direction/button key is enabled by
-bit 4/5, respectively), however, because of switch bounce, one or more High to Low
-transitions are usually produced when pressing and releasing a key.
+from High to Low. This happens when a button is
+pressed (provided that the action/direction button is enabled by
```
Comment (ISSOtm) [inline content/Joypad_Input.md:35]: Please use plural "button**s are**"

**Review by ISSOtm — CHANGES_REQUESTED** [review]

**Comment by ISSOtm** [comment]
@avivace What do we think about the name changes?

File: content/Joypad_Input.md (hunk 1 above)
Comment (AlvaroBurnett) [reply content/Joypad_Input.md]: Fixed

Comment (AlvaroBurnett) [reply content/Joypad_Input.md]: Fixed

**Review by ISSOtm — APPROVED** [review]

File: content/Joypad_Input.md [hunk 3]
```diff
@@ -30,18 +30,18 @@ to the SNES. See SGB description for details.
...
+transitions are usually produced when pressing a button.
 
 ### Using the Joypad Interrupt
 
 It's practically useless for programmers. Even when selecting both
-buttons and direction keys simultaneously it still cannot recognize all
-keystrokes, because in that case a bit might be already held low by a
-button key, and pressing the corresponding direction key would thus
+action and direction buttons simultaneously, it still cannot recognize all
+presses, because in that case a bit might be already held Low by an
```
Comment (avivace) [inline content/Joypad_Input.md:43]: Keystrokes is clearer than presses in my opinion

**Review by avivace — CHANGES_REQUESTED** [review]

File: content/Joypad_Input.md (hunk 3 above)
Comment (AlvaroBurnett) [reply content/Joypad_Input.md:43]: And "button presses"?

Comment (avivace) [reply content/Joypad_Input.md:43]: "each button press" is probably better

Comment (AlvaroBurnett) [reply content/Joypad_Input.md]: I have rewritten it to avoid the issue

**Review by ISSOtm — CHANGES_REQUESTED** [review]

File: content/Joypad_Input.md [hunk 4]
```diff
@@ -30,18 +30,18 @@ to the SNES. See SGB description for details.
...
+transitions are usually produced when pressing a button.
 
 ### Using the Joypad Interrupt
 
-It's practically useless for programmers. Even when selecting both
-buttons and direction keys simultaneously it still cannot recognize all
-keystrokes, because in that case a bit might be already held low by a
-button key, and pressing the corresponding direction key would thus
+This interrupt is useful to identify button presses if we have only selected
+either action (bit 5) or direction (bit 4), but not both,
```
[the remark this answers is not in the document]
Comment (AlvaroBurnett) [reply content/Joypad_Input.md]: Fixed

**Review by ISSOtm — APPROVED** [review]

**Review by avivace — APPROVED** [review]

**Pull request merged**